conf/conf.go

```diff
@@ -232,3 +256,43 @@ func GetSmtpCredentials() *SmtpCredentials {
 
 	return smtpCred
 }
+
+// SmtpCheck tests whether a connection to the specified smtp server can be established
+// with the provided credentials and will panic if it cannot.
+func SmtpCheck() {
+	cred := GetSmtpCredentials()
+	if cred.Mode == "skip" || cred.Mode == "print" {
+		return
+	}
+
+	addr := cred.Host + ":" + strconv.Itoa(cred.Port)
+	auth := smtp.PlainAuth("", cred.Username, cred.Password, cred.Host)
+
+	netCon, err := net.DialTimeout("tcp", addr, time.Second*10)
```

Comment:
Is it necessary to close the connection? If yes, calling `defer netCon.Close()` if `err == nil`.